src/core/game/game_processor/game_processor.pve_classic.ts

```diff
@@ -0,0 +1,309 @@
+import { PlayerCardsArea, PlayerId, PlayerInfo, PlayerRole } from 'core/player/player_props';
+import { StandardGameProcessor } from './game_processor.standard';
+import { Player } from 'core/player/player';
+import { Sanguosha } from '../engine';
+import {
+  CardDrawReason,
+  CardMoveArea,
+  CardMoveReason,
+  ClientEventFinder,
+  EventPacker,
+  GameEventIdentifiers,
+  ServerEventFinder,
+} from 'core/event/event';
+import { Character } from 'core/characters/character';
+import { GameEventStage, PlayerDiedStage } from '../stage_processor';
+import { TranslationPack } from 'core/translations/translation_json_tool';
+import { GameMode } from 'core/shares/types/room_props';
+import { Functional } from 'core/shares/libs/functional';
+import { CardId } from 'core/cards/libs/card_props';
+import { VirtualCard } from 'core/cards/card';
+import { PveClassicAi } from 'core/skills/game_mode/pve/pve_classic_ai';
+import { MarkEnum } from 'core/shares/types/mark_list';
+import { Algorithm } from 'core/shares/libs/algorithm';
+import { PveClassicGuYong } from 'core/skills';
+
+export class PveClassicGameProcessor extends StandardGameProcessor {
+  private level: number = 0;
+  private mark_list: MarkEnum[] = [];
```

Comment:
recommend to consist of one naming styles all the time 
```suggestion
  private markList: MarkEnum[] = [];
```